src/provider_misc.jl

```diff
@@ -311,7 +311,9 @@ end
 
 
 function process(r::JSONRPC.Request{Val{Symbol("julia/toggle-lint")},TextDocumentIdentifier}, server)
-    server.documents[r.uri]._runlinter != server.documents[r.uri]._runlinter
+    # TODO Isn't this always false? I think this is meant to toggle things
+    # i.e. use assignment rather than a comparison here, right?
+    server.documents[filepath_from_uri(r.uri)]._runlinter != server.documents[filepath_from_uri(r.uri)]._runlinter
```

Comment:
@ZacLN This is just a bug, right? It should simply flip the value of ``_runlinter``?

Reply:
yup def a bug

Reply:
Alright, I fixed this with a commit, if you could take a brief look at that as well, it would be great. Is part of this PR.